CryptoTechReminderSystem/DomainObject/PostMessageResponse.cs

```diff
@@ -0,0 +1,31 @@
+using System;
+
+namespace CryptoTechReminderSystem.DomainObject
+{
+    public class PostMessageResponse<TSuccess, TException>
```

Comment:
I would use TError because these aren't actually exceptions

Reply:
I had TError initially and it only had a message property so the System.Exception seemed a good alternative

Reply:
Also, I am not sure if this is a domain or boundary object?